test/test_torch.py

```diff
@@ -5552,6 +5555,11 @@ def test_is_signed(self):
         self.assertEqual(torch.CharTensor(5).is_signed(), True)
         self.assertEqual(torch.FloatTensor(5).is_signed(), True)
         self.assertEqual(torch.HalfTensor(10).is_signed(), True)
+        W = torch.rand(2).float()
```

Comment:
why is this tested here instead of one of the quantized tests?  I'm not even sure what the answers of these should be.  Can we just pull these out and file an issue that quantized dtypes need to be actually tested?